src/main/java/org/osc/sdk/manager/element/ManagerSecurityGroupInterfaceElement.java

```diff
@@ -34,10 +36,21 @@
      */
     String getName();
 
-    /**
-     * @return the identifier of this policy defined by the security manager
-     */
-    String getPolicyId();
+	/**
+	 * Provide manager security group id if the manager is supporting policy
+	 * mapping and security group.
+	 *
+	 * @return the manager security group identifier
+	 */
+	String getManagerSecurityGroupId();
+
+	/**
+	 * Provide manager policy id in case of IP/MAC or encapsulation tag in case
```

Comment:
Providers the identifier of the policies supported by the manager. If the appliance uses encapsulation tags this should be the tags supported by the manager.